workstreams/supply-chain-maturity/README.md

```diff
@@ -0,0 +1,44 @@
+# Supply Chain Maturity Workstream README
+
+This is a workstream within the CDF SIG Software Supply Chain.
+
+The forming of this workstream was suggested at a [recent SIG Software Supply
+Chain
+meeting](https://github.com/cdfoundation/sig-software-supply-chain/blob/main/docs/meetings.md#August-25-2022)
+
+## Scope
+
+The workstream will cover CICD beyond build and deploy.
+
+## Mission
+
+The workstream seeks to define industry standards for Supply Chain Maturity that
+augment [SLSA](http://slsa.dev).  Where SLSA covers the supply chain from code
+through artifact, maturity covers the artifact lifecycle, including deployments,
+rollouts, testing, rollbacks, and more.
+
+The workstream seeks to define an industry standard Code Health Project Score
+("CHiPS") that augment [SLSA](http://slsa.dev).
```

Comment:
I liked the acronym a lot but @oleg-nenashev pointed out that there is a project named [CHIPS Alliance](https://chipsalliance.org/) under the Linux Foundation and it might confuse people.

Reply:
updated to reflect this tragic news